whitesource/client.py

```diff
@@ -171,7 +177,7 @@ def __init__(
         self.extension_endpoint = extension_endpoint
         self.wss_api_endpoint = wss_api_endpoint
 
-    def post_component(self):
+    def ws_component(self):
```

Comment:
same here: please choose a name that expresses the (highlevel) operation, rather than the technical implementation-detail of the used http-"method" (ws is not actually a http-method, as you correctly noted yesterday)